package-lock.json

```diff
@@ -1,7493 +1,8 @@
 {
   "name": "bot-v2",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
There seems to be a lot of deletions here for a  version bump. When you run npm install locally does it have any un-staged changes in the package-lock.json?

Reply:
I just ran `npm i` while in my branch and it says it's up to date. I did notice that I don't have a "packages" key, but I have a "dependencies" key now.